src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1641,354 +1636,229 @@ namespace ScheduleManager {
-            //      DO Hr=1,24
-            //        SCount=1
-            //        CurMinute=MinutesPerTimeStep
-            //        DO TS=1,NumOfTimeStepInHour
-            //          DaySchedule(Count)%TSValue(Hr,TS)=SUM(MinuteValue(Hr,SCount:CurMinute))/REAL(MinutesPerTimeStep,r64)
-            //          SCount=CurMinute+1
-            //          CurMinute=CurMinute+MinutesPerTimeStep
-            //        ENDDO
-            //      ENDDO
-            //    ELSE
-            //      DO Hr=1,24
-            //        CurMinute=MinutesPerTimeStep
-            //        DO TS=1,NumOfTimeStepInHour
-            //          DaySchedule(Count)%TSValue(Hr,TS)=MinuteValue(Hr,CurMinute)
-            //          Curminute=CurMinute+MinutesPerTimeStep
-            //        ENDDO
-            //      ENDDO
-            //    ENDIF
+        // schedule values have been filled into the columnarData.vals vectors.
 
-            std::string contextString = CurrentModuleObject + "=\"" + Alphas(1) + "\", " + cAlphaFields(3) + ": ";
+        // skip rows when needed - potentially unnecessary, since
+        for (auto &schedule : state.dataScheduleMgr->columnarData) {
+            if (schedule.rowsToSkip != -1) {
+                schedule.vals.erase(schedule.vals.begin(), schedule.vals.begin() + schedule.rowsToSkip);
+            }
+        }
 
-            CheckForActualFileName(state, Alphas(3), FileExists, state.files.TempFullFileName.fileName, contextString);
+        for (const PreProcessedColumn &schedule : state.dataScheduleMgr->columnarData) {
+            rowLimitCount = (schedule.numHourlyValues * 60) / schedule.MinutesPerItem;
+            if (schedule.numerrors > 0) {
+                ShowWarningError(state,
+                                 format("{}{}=\"{}\" {} records had errors - these values are set to 0.",
+                                        RoutineName,
+                                        CurrentModuleObject,
+                                        schedule.name,
+                                        schedule.numerrors));
+                ShowContinueError(state, "Use Output:Diagnostics,DisplayExtraWarnings; to see individual records in error.");
+            }
+            if (schedule.rowCnt < rowLimitCount) {
+                ShowWarningError(state,
+                                 format("{}{}=\"{}\" less than {} hourly values read from file.",
+                                        RoutineName,
+                                        CurrentModuleObject,
+                                        schedule.name,
+                                        schedule.numHourlyValues));
+                ShowContinueError(state,
+                                  format("..Specified Number of Hourly Values={} Actual number of hourly values included={}",
+                                         schedule.numHourlyValues,
+                                         (rowCnt * 60) / MinutesPerItem));
+            }
+        }
```

Comment:
Warnings if the number of rows specified in IDF is not equal to the number of rows processed.